app/views/shared/_focus_trap_script.html.erb

```diff
@@ -0,0 +1,66 @@
+<script>
+    function getFocusTrapToggle(containerSelector, focusTrapSelector, activatorSelector) {
```

Comment:
This whole chunk of JS I'm going to need in `app > assets > javascripts > utilities` but it doesn't seem to then be available in the admin pages. Is there another way to pull in JS from the assets folder to these pages?

Reply:
Does this work? `//= require utilities/getFocusTrapToggle` ?

Reply:
(update) - I'm going to look at tackling the modals in the `admin` area with the stimulus `modal_controller`